contracts/settling_game/utils/log.cairo

```diff
@@ -0,0 +1,48 @@
+%builtins output range_check
```

Comment:
I'll just presume this whole thing works 😂  Where did you find it?

Can you please clean up the imports a little and remove the `%builtins` directive?

Reply:
this was a gyoza creation